shared/Features/FeatureFlags.cs

```diff
@@ -8,13 +8,14 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.Features
     public class FeatureFlags : IFeatureFlags
     {
         private readonly IConfiguration _config;
-        private const string APPLY_2019 = "FEATURE_APPLY_2019";
 
         public FeatureFlags(IConfiguration config)
         {
             _config = config;
         }
-        public bool Apply2019 => ShouldShow(APPLY_2019);
+
+        public bool Apply2019 => ShouldShow("FEATURE_APPLY_2019");
```

Comment:
Delete this?

Reply:
https://trello.com/c/4XxSadm2/475-remove-apply-feature-flag

Reply:
I'm trying to keep each PR tight in scope, they have a habit of getting out of hand!